pkg/migrations/op_add_column.go

```diff
@@ -450,3 +452,53 @@ func (w ColumnSQLWriter) Write(col Column) (string, error) {
 	}
 	return sql, nil
 }
+
+func (o *OpAddColumn) Create() {
```

Comment:
I think all of the implementations of `Create` and helper functions should be moved to a `pterm_create.go` file, as they aren't related to the mechanics of the operation and add quite a lot of `pterm` boilerplate code to some operations.